### PR TITLE
Add CommissionedFabrics and SupportedFabrics attribute to opCreds cluster

### PR DESCRIPTION
**Pull request by shana-apple**
#### Problem
Add CommissionedFabrics and SupportedFabrics attribute to opCreds cluster
#### Change overview
Add CommissionedFabrics and SupportedFabrics attribute to opCreds cluster
Update CommissionedFabrics when number of fabrics is updated in fabrics list.

### CODE REVIEW

**Review by woody-apple — APPROVED** [review]

File: src/app/clusters/operational-credentials-server/operational-credentials-server.cpp [hunk 1]
```diff
@@ -140,7 +148,12 @@ CHIP_ERROR writeAdminsIntoFabricsListAttribute()
     emberAfPrintln(EMBER_AF_PRINT_DEBUG, "OpCreds: Stored %" PRIu32 " admins in fabrics list attribute.", fabricIndex);
```
Comment (bzbarsky-apple) [inline src/app/clusters/operational-credentials-server/operational-credentials-server.cpp:148]:
```suggestion
    emberAfPrintln(EMBER_AF_PRINT_DEBUG, "OpCreds: Stored %" PRIu8 " admins in fabrics list attribute.", fabricIndex);
```

File: src/app/clusters/operational-credentials-server/operational-credentials-server.cpp [hunk 2]
```diff
@@ -140,7 +148,12 @@ CHIP_ERROR writeAdminsIntoFabricsListAttribute()
...
     if (writeFabricAttribute((uint8_t *) &fabricIndex) != EMBER_ZCL_STATUS_SUCCESS)
```
Comment (bzbarsky-apple) [inline src/app/clusters/operational-credentials-server/operational-credentials-server.cpp:149]:
```suggestion
    if (writeFabricAttribute(&fabricIndex) != EMBER_ZCL_STATUS_SUCCESS)
```

File: src/app/zap-templates/zcl/data-model/chip/operational-credentials-cluster.xml [hunk 3]
```diff
@@ -32,6 +32,8 @@ limitations under the License.
     <description>This cluster is used to add or remove Operational Credentials on a Commissionee or Node, as well as manage the associated Fabrics.</description>
 
     <attribute side="server" code="0x0001" define="FABRICS" type="ARRAY" entryType="FabricDescriptor" length="254" writable="false" optional="false">fabrics list</attribute>
+    <attribute side="server" code="0x0002" define="SUPPORTED_FABRICS" type="INT8U" min="0x00" max="0xFF" writable="false" optional="false">supported fabrics</attribute>
+    <attribute side="server" code="0x0003" define="COMMISSIONED_FABRICS" type="INT8U" min="0x00" max="0xFF" writable="false" optional="false">commissioned fabrics</attribute>
```
Comment (bzbarsky-apple) [inline src/app/zap-templates/zcl/data-model/chip/operational-credentials-cluster.xml:36]: Shouldn't these also be added to .zap files somewhere?

File: src/controller/data_model/gen/CHIPClusters.cpp [hunk 4]
```diff
@@ -21,8 +21,8 @@
 #include <cstdint>
```
Comment (bzbarsky-apple) [inline src/controller/data_model/gen/CHIPClusters.cpp:21]: The changes to this file are a bit unexpected... and are failing the ZAP CI.

**Review by bzbarsky-apple — CHANGES_REQUESTED** [review]

**Comment by mspang** [comment]
@shana-apple This didn't get sent out because it's failing restyle..

**Comment by shana-apple** [comment]
Closing and switching to: https://github.com/shana-apple/connectedhomeip/pull/new/sazria/commissionedFabrics

**Pull request closed without merging**